core/src/main/scala/org/apache/spark/rpc/netty/Dispatcher.scala

```diff
@@ -106,7 +106,7 @@ private[netty] class Dispatcher(nettyEnv: NettyRpcEnv) extends Logging {
     val iter = endpoints.keySet().iterator()
     while (iter.hasNext) {
       val name = iter.next
-      postMessage(name, message, (e) => logWarning(s"Message $message dropped.", e))
+      postMessage(name, message, (e) => logWarning(s"Message $message dropped."))
```

Comment:
Could you update the message as `s"Message $message dropped. ${e.getMessage}"`? It's worth to add the exception message here since there are two types of exceptions.